pulse_actions/publisher.py

```diff
@@ -16,10 +16,10 @@
 
 
 class ExperimentalPublisher(GenericPublisher):
-    def __init__(self, **kwargs):
+    def __init__(self, user, **kwargs):
         super(ExperimentalPublisher, self).__init__(
             PulseConfiguration(**kwargs),
-            'exchange/adusca/experiment',
+            'exchange/%s/pulse_actions' % user,
```

Comment:
I think we still need to pass user as part of kwargs. This way kwargs are just {password: password} [1] and we need it to be {user: user, password: password}.

```
>>> def f(user, **kwargs):
...     print kwargs
... 
>>> f(user='user', password='1')
{'password': '1'}
>>> 
```